tasks/internal/filter-already-released-advisory-images-task/filter-already-released-advisory-images-task.yaml

```diff
@@ -138,6 +146,23 @@ spec:
 
           echo "Comparing against: $ADVISORY_FILE"
 
+          # Track this advisory as the latest (since they're sorted by modification time, newest first)
+          if [[ -z "$LATEST_ADVISORY_URL" ]]; then
+            # Extract advisory details for URL generation
+            ADVISORY_TYPE=$(yq -r '.spec.type' "$ADVISORY_FILE")
+            ADVISORY_NAME=$(yq -r '.metadata.name' "$ADVISORY_FILE")
```

Comment:
there are two new/additional yq calls per advisory were added by this commit. maybe it's worth to reduce number from 3 (image, type, name) to 1 call per advisory by extracting all advisory data in a single yq call. something like this:

`
          ADVISORY_DATA=$(yq -o=json '{
            "images": (.spec.content.images // []),
            "type": .spec.type,
            "name": .metadata.name
          }' "$ADVISORY_FILE")
          
          # Parse the extracted data
          ADVISORY_IMAGES=$(echo "$ADVISORY_DATA" | jq -r '.images')
          ADVISORY_TYPE=$(echo "$ADVISORY_DATA" | jq -r '.type')
          ADVISORY_NAME=$(echo "$ADVISORY_DATA" | jq -r '.name')
          
          echo "$ADVISORY_IMAGES" > "$EXISTING_CONTENT"
`